### ISSUE TITLE
Error AST 3.2.0 with PlotSquared v4.

### ISSUE THREAD

**Issue by x1p** [St3venAU/ArmorStandTools#18]
```
[00:25:43 INFO]: [ArmorStandTools] Enabling ArmorStandTools v3.2.0
[00:25:43 INFO]: [ArmorStandTools] Loading support for v1_13_R2
[00:25:43 WARN]: java.lang.NoClassDefFoundError: com/intellectualcrafters/plot/api/PlotAPI
[00:25:43 WARN]: at com.gmail.St3venAU.plugins.ArmorStandTools.PlotSquaredHook.<init>(PlotSquaredHook.java:17)
[00:25:43 WARN]: at com.gmail.St3venAU.plugins.ArmorStandTools.Config.reloadMainConfig(Config.java:146)
[00:25:43 WARN]: at com.gmail.St3venAU.plugins.ArmorStandTools.Config.reload(Config.java:57)
[00:25:43 WARN]: at com.gmail.St3venAU.plugins.ArmorStandTools.Config.reload(Config.java:53)
[00:25:43 WARN]: at com.gmail.St3venAU.plugins.ArmorStandTools.Main.onEnable(Main.java:41)
[00:25:43 WARN]: at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:254)
[00:25:43 WARN]: at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:325)
[00:25:43 WARN]: at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405)
[00:25:43 WARN]: at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:441)
[00:25:43 WARN]: at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:355)
[00:25:43 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:613)
[00:25:43 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:575)
[00:25:43 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:435)
[00:25:43 WARN]: at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:316)
[00:25:43 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:791)
[00:25:43 WARN]: at java.lang.Thread.run(Thread.java:748)
[00:25:43 WARN]: Caused by: java.lang.ClassNotFoundException: com.intellectualcrafters.plot.api.PlotAPI
[00:25:43 WARN]: at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[00:25:43 WARN]: at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:158)
[00:25:43 WARN]: at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:104)
[00:25:43 WARN]: at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[00:25:43 WARN]: at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[00:25:43 WARN]: ... 16 more
[00:25:43 WARN]: [ArmorStandTools] PlotSquared plugin was found, but there was an error initializing PlotSquared support enabled.
[00:25:43 INFO]: [ArmorStandTools] WorldGuard plugin found. WorldGuard support enabled.
```
Using:
Paper 1.13.2 build 488
ArmorStandTools-3.2.0
PlotSquared-Bukkit-4.18.12.28-3156741-2126

**Comment by x1p** [comment]
> It's an issue with armorstandtools. It needs to update to the new P2 API.
Could you please investigate?

**Comment by x1p** [comment]
Could you please investigate?
A month has passed...

**Comment by xion87** [comment]
i noticed with this error, people are able to abuse the armor stand at spawn adn doing shitty thing, for the other features is working fine

**Comment by Kryniowesegryderiusz** [comment]
@St3venAU

**Comment by x1p** [comment]
I think it's solved with v 3.3.0. so this issue can be closed I think
